src/credentials/auth.ts

```diff
@@ -680,10 +682,16 @@ export class Auth implements AuthService, ConnectionManager {
                 cause: this.#validationErrors.get(id),
             })
         }
-        // TODO: cancellable notification?
         if (previousState === 'valid') {
+            const timeout = new Timeout(60000)
+            this.#invalidCredentialsTimeouts.set(id, timeout)
+
             const message = localize('aws.auth.invalidConnection', 'Connection is invalid or expired, login again?')
-            const resp = await vscode.window.showInformationMessage(message, localizedText.yes, localizedText.no)
+            const resp = await Promise.race([
```

Comment:
should we prefer `Promise.any` ?

Reply:
We're trying to bypass the prompt here so `Promise.race` is preferred. If we could programmatically close the notification this would be so much cleaner :/